fluent-validator/src/main/java/com/baidu/unbiz/fluentvalidator/ValidatorContext.java

```diff
@@ -36,6 +38,19 @@
      */
     public ValidationResult result;
 
+    public boolean validateWith(Function<FluentValidator, FluentValidator> configuration) {
+        ComplexResult localResult = configuration.apply(FluentValidator.checkAll())
```

Comment:
Hopefully I will provide a SNAPSHOT version and explaination for you to use tomorrow, please wait some time to give it a try.